tests/src/test/java/tachyon/client/LocalBlockInStreamIntegrationTest.java

```diff
@@ -94,7 +107,6 @@ public void readTest1() throws IOException, TachyonException {
         Assert.assertEquals(cnt, k);
         Assert.assertTrue(BufferUtils.equalIncreasingByteArray(k, ret));
         is.close();
-        Assert.assertTrue(sFileSystem.getStatus(uri).getInMemoryPercentage() == 100);
```

Comment:
It doesn't make sense for it to be in memory here, there is no guarantee. The second check makes sense because we read with cache mode.